src/ContainerRestoreModal.jsx

```diff
@@ -28,7 +28,7 @@ const ContainerRestoreModal = ({ containerWillRestore, onAddNotification }) => {
             ignoreStaticMAC,
         })
                 .catch(ex => {
-                    const error = cockpit.format(_("Failed to restore container $0"), containerWillRestore.Names);
+                    const error = cockpit.format(_("Failed to restore container $0"), containerWillRestore.Name);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1350-20230712-152457-eee84e1f-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)